ttter/accounts/views.py

```diff
@@ -39,3 +43,120 @@ def form_valid(self, form):
 
     def form_invalid(self, form):
         return render(self.request, "accounts/create.html", {"form": form})
+
+
+class UserListView(LoginRequiredMixin, ListView):
+    template_name = "accounts/user_list.html"
+    model = User
+
+    def get_queryset(self):
+        return User.objects.values_list("username", flat=True)
+
+
+class UserPage(LoginRequiredMixin, TemplateView):
+    template_name = "accounts/user_page.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        request_user = self.request.user
+        user = get_object_or_404(User, username=self.kwargs.get("username"))
+        post_item = Tweet.objects.filter(user=user)
+        context["is_followed"] = FriendShip.objects.filter(
+            followee=request_user, follower=user
+        ).exists()
+        context["request_user"] = request_user
+        context["user"] = user
+        context["post_item"] = post_item
+        context["followee_count"] = user.followee.count()
+        context["follower_count"] = user.follower.count()
+        return context
+
+
+class FollowerListView(LoginRequiredMixin, TemplateView):
+    template_name = "accounts/follower_list.html"
+
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data(**kwargs)
+        self.user = get_object_or_404(User, username=self.kwargs["username"])
+
+        ctx["followers"] = [
+            follower.followee
+            for follower in FriendShip.objects.select_related("follower").filter(
+                follower=self.user
+            )
+        ]
+        return ctx
+
+
+class FollowingListView(LoginRequiredMixin, TemplateView):
+    template_name = "accounts/following_list.html"
+
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data(**kwargs)
+        self.user = get_object_or_404(User, username=self.kwargs["username"])
```

Comment:
```suggestion
        user = get_object_or_404(User, username=self.kwargs["username"])
```
ここもselfいらないです。FollowerListViewも同様

Reply:
了解です！